src/main/java/com/beerjournal/breweriana/image/item/ImageItemController.java

```diff
@@ -0,0 +1,52 @@
+package com.beerjournal.breweriana.image.item;
+
+import com.mongodb.gridfs.GridFSDBFile;
+import lombok.RequiredArgsConstructor;
+import org.bson.types.ObjectId;
+import org.springframework.core.io.InputStreamResource;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.util.Set;
+
+@RestController
+@RequestMapping("/api/images/item")
+@RequiredArgsConstructor
+public class ImageItemController {
+
+    private final ImageItemService imageItemService;
+
+    @GetMapping
```

Comment:
Hmm, I think endpoint I've proposed is better. Because adding an image is similar to PUT on item which is `/api/users/items/{itemId}`